code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -796,6 +803,8 @@
 	construction_time = 120
 	category = list("Cyborg Upgrade Modules")
 
+//SKYRAT EDIT REMOVAL BEGIN
+/*
 /datum/design/borg_transform_clown
 	name = "Cyborg Upgrade (Clown Model)"
```

Comment:
again, clowns exist, clowns are not inherently lrp by mere existence, if anything, lock it behind vet crew